src/client/ControlPanel/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import * as d3 from 'd3'
+import * as d3 from 'd3/dist/d3'
```

Comment:
`parcel` would not import this properly in its previous form, it would be undefined at runtime.